psi4/share/cmake/psi4/psi4Config.cmake.in

```diff
@@ -74,7 +74,7 @@ set(psi4_FOUND TRUE)
 set(PSI4_FOUND TRUE)
 
 function(add_psi4_plugin TARGET SOURCES)
-    # remove ${TARGET} from ${ARGV} to use ${ARGV} as ${SOURCES}
```

Comment:
The old https://github.com/psi4/psi4/blob/master/psi4/psi4-config.in has a lot of (once-useful) definitions. It hasn't been renovated since CMR-KtB-HRW-INV